ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContactCollectionView)
+open class ContactCollectionView: UICollectionView {
+    struct Constants {
+        static let sideInset: CGFloat = 16.0
+        static let extraSmallContentContactHeight: CGFloat = 115.0
+        static let smallContentContactHeight: CGFloat = 117.0
+        static let mediumContentContactHeight: CGFloat = 118.0
+        static let largeContentContactHeight: CGFloat = 121.0
+        static let extraLargeContentContactHeight: CGFloat = 125.0
+        static let extraExtraLargeContentContactHeight: CGFloat = 129.0
+        static let extraExtraExtraLargeContentContactHeight: CGFloat = 135.0
+    }
+
+    let layout: ContactCollectionViewLayout
+    @objc public weak var contactCollectionDelegate: UICollectionViewDelegate?
+
+    @objc public var contactList: [PersonaData] = [] {
+        didSet {
+            if (oldValue.count == 0 && contactList.count > 0) || (oldValue.count > 0 && contactList.count == 0) {
+                updateHeightConstraint()
+            }
+        }
+    }
+
+    @objc public init() {
```

Comment:
Should the initializer take a [PersonaData] array for convenience?

Reply:
I don't think we need the initializer to take in a `[PersonaData]` in `init()` since they can set the `contactList` 
variable. We also discussed this in the engineering plan with Miguel and mentioned that it (`[ContactView]`) shouldn't be part of the initializer. Do you think we should still include a `[PersonaData]` array in the `init()`?

Reply:
It would be mainly for convenience since anyone creating a ContactCollection would want to pass it PersonaDatas. Up to you if you want to go that direction or not. This is mostly an API design consideration.